config/development.yml.erb

```diff
@@ -5,6 +5,7 @@ dashboard_enable_pegasus: true
 
 firebase_name: cdo-v3-dev
 # provide a unique path for firebase channels data for development, to avoid conflicts in channel ids.
+firebase_shared_secret: !Secret
```

Comment:
you shouldn't need this, since the same thing is specified in config.yml.erb

Reply:
`development.yml.erb` has `clear_secrets` at the top, so it is needed. The general rationale, I think, is that development environments shouldn't have access to secrets by default, and to make it an intentional choice to make them available. In this case I think it's reasonable since these credentials should be for read-only access (@ajpal to confirm).

Reply:
Yes, the permissions for this table are read-only. It's only updated using the Ruby scripts which, if I understand correctly, run with different permissions anyways. Adding the secret to the dev environment is just necessary to be able to see the shared data in app lab while developing